### ISSUE TITLE
dropout rate value for best model

### ISSUE THREAD

**Issue by jbwoillard** [r-tensorflow/autokeras#17]
Dear Juan Cruz, I was wondering how to access the dropout rate for dropout layers selected for the best model obtained with the export_model function.
Thanks again for autokeras.

**Comment by jcrodriguez1989** [comment]
Ha, thanks for posting here @jbwoillard !
Suppose your autokeras model is named `autokeras_model`

Get your best trained model with:
`> keras_model <- export_model(autokeras_model)`

Now print your model, and pay special attention to the first column. This first column shows each layer, with its name first, and between parentheses the layer type.
```r
> keras_model
Model
Model: "model"

Layer (type)                                  Output Shape                              Param #         
========================================================================================================
input_1 (InputLayer)                          [(None, 1)]                               0               
text_vectorization (TextVectorization)        (None, 20000)                             0               
dense (Dense)                                 (None, 64)                                1280064         
batch_normalization (BatchNormalization)      (None, 64)                                256             
re_lu (ReLU)                                  (None, 64)                                0               
dropout (Dropout)                             (None, 64)                                0               
regression_head_1 (Dense)                     (None, 1)                                 65              
========================================================================================================
Total params: 1,300,385
Trainable params: 1,280,257
Non-trainable params: 20,128
```

So, the layer you want is named `"dropout"`. Let's get it, and then extract its rate:
```r
> dropout_layer <- keras_model %>% get_layer("dropout")
> names(dropout_layer)
 [1] "activity_regularizer"     "add_loss"                 "add_metric"              
 [4] "add_update"               "add_variable"             "add_weight"              
 [7] "apply"                    "build"                    "built"                   
[10] "call"                     "compute_mask"             "compute_output_shape"    
[13] "compute_output_signature" "count_params"             "dtype"                   
[16] "dynamic"                  "from_config"              "get_config"              
[19] "get_input_at"             "get_input_mask_at"        "get_input_shape_at"      
[22] "get_losses_for"           "get_output_at"            "get_output_mask_at"      
[25] "get_output_shape_at"      "get_updates_for"          "get_weights"             
[28] "inbound_nodes"            "input"                    "input_mask"              
[31] "input_shape"              "input_spec"               "losses"                  
[34] "metrics"                  "name"                     "name_scope"              
[37] "noise_shape"              "non_trainable_variables"  "non_trainable_weights"   
[40] "outbound_nodes"           "output"                   "output_mask"             
[43] "output_shape"             "rate"                     "seed"                    
[46] "set_weights"              "stateful"                 "submodules"              
[49] "supports_masking"         "trainable"                "trainable_variables"     
[52] "trainable_weights"        "updates"                  "variables"               
[55] "weights"                  "with_name_scope" 
> dropout_layer$rate
0.5
```

I hope, this works for you, let me know if you have any further concern.
